README.md

```diff
@@ -28,12 +28,6 @@ From the Kubernetes documentation on [Encrypting Secret Data at Rest]:
 
 💡 Make sure you have a Kubernetes cluster version 1.10 or later, the minimum version that is supported by KMS Plugin for Key Vault.
 
-### 🎁 aks-engine
-
-[AKS Engine] creates customized Kubernetes clusters on Azure.
-
-Follow the AKS Engine documentation about [Azure Key Vault Data Encryption] and refer to the [example cluster configuration] to create a Kubernetes cluster with KMS Plugin for Key Vault automatically configured. Once the cluster is running, there will be an Azure Key Vault containing a new key in the same resource group as the cluster.
-
 ### Azure Kubernetes Service (AKS)
 
 Azure Kubernetes Service ([AKS]) creates managed, supported Kubernetes clusters on Azure.
```

Comment:
I think we should also point to AKS-KMS preview doc in this section for users to try it out.
https://docs.microsoft.com/en-us/azure/aks/use-kms-etcd-encryption

Currently links in this section points to general container security

Reply:
Updated!

Reply:
Thanks @salaxander. One more small nit. There are also references of AKS-Engine in manual installation doc. We should remove that too.

https://github.com/Azure/kubernetes-kms/blob/master/docs/manual-install.md#obtaining-the-id-of-the-cluster-managed-identityservice-principal

Reply:
One more update pushed, thanks for the review!